flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/api/FlinkLocalCluster.java

```diff
@@ -48,12 +49,10 @@
 	private static final Logger LOG = LoggerFactory.getLogger(FlinkLocalCluster.class);
 
 	/** The flink mini cluster on which to execute the programs */
-	private final FlinkMiniCluster flink;
+	private FlinkMiniCluster flink;
 
 
 	public FlinkLocalCluster() {
-		this.flink = new LocalFlinkMiniCluster(new Configuration(), true, StreamingMode.STREAMING);
-		this.flink.start();
 	}
```

Comment:
Lazy startup of the flink cluster. This enables us to set the right number of task slots for the job.